person.rb

```diff
@@ -21,6 +23,10 @@ def correct_name
     name
   end
 
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
+
```

Comment:
Nice job implementing a method for adding rental in the person class as requested by the previous reviewer😉. Kindly note that you're making a little mistake by using exactly the same method for adding rentals in both the person and the book class😉. The arguments for the method here should be the book and the date😉 since `self` already represents the person😉. Kindly make use of the suggestion below to make the necessary change👍🏼.

```suggestion
  def add_rental(book, date)
    Rental.new(date, self, book)
  end

```